src/SFML.Graphics/Text.cs

```diff
@@ -281,14 +287,14 @@ public Vector2f FindCharacterPos(uint index)
 
         ////////////////////////////////////////////////////////////
         /// <summary>
-        /// Get the local bounding rectangle of the entity.
-        ///
-        /// The returned rectangle is in local coordinates, which means
-        /// that it ignores the transformations (translation, rotation,
-        /// scale, ...) that are applied to the entity.
-        /// In other words, this function returns the bounds of the
-        /// entity in the entity's coordinate system.
+        /// Get the LOCAL bounding <see cref="FloatRect"/> of the text.
         /// </summary>
+        /// <remarks>
+        /// The returned <see cref="FloatRect"/> is in LOCAL coordinates.
+        /// <para>Transformations (Translation, Rotation, Scale) ARE NOT applied to the entity.</para>
```

Comment:
I think `are not` (lower case) is sufficient here

Reply:
LOCAL and ARE NOT were capitalized to emphasize the differences between GetLocalBounds() and GetGlobalBounds().  Having emphasis for details like this isn't out of the ordinary from my experience, but if we want to remove that emphasis, I completely understand and can make that change.

Reply:
Yeah not a fan and I think it's not necessary.

Reply:
Roger.  I'll go ahead and nix these from both then,